lib/memcached.js

```diff
@@ -913,7 +913,7 @@ Client.config = {
     value = Utils.escapeValue(value);
 
     length = Buffer.byteLength(value);
-    if (length > this.maxValue) {
+    if (length + fullkey.length + 71 > this.maxValue) {
```

Comment:
Something to keep in mind is that memcache clients can also talk to Couchbase servers. I don't know what the limitations are there? In any case, I think a code comment or well chosen constant name would be nice.